app/views/events/badges.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1><%= t('events.badges.title') %>'<%= @event.name %>'</h1>
+<h3><%= t('events.badges.participants') %></h3>
+
+<%= form_for event_badges_path(@event), :html => { :class => "form-horizontal event" } do |f| %>
+<table>
+  <tr>
+    <th>Name</th>
+    <th><%= t('events.badges.printBoolean') %></th>
+  </tr>
+
+  <% @participants.each do |participant| %>
+      <tr>
+        <td>
+          <%= participant.name %>
+        </td>
+        <td>
+          <%= f.check_box_tag "#{participant.id}_print_#{participant.name}"%>
```

Comment:
You can pass the check_box_tag function a `:value` hash. That way you don't have to parse the key to get the name.